src/CMakeLists.txt

```
@@ -19,7 +19,7 @@ set(hueplusplus_SOURCES
     StateTransaction.cpp
     TimePattern.cpp
     UPnP.cpp
-    Utils.cpp)
+    Utils.cpp "BridgeConfig.cpp")
```

This should be corrected
```suggestion
    Utils.cpp
    BridgeConfig.cpp)
```